source/billing/credit-card.rst

```diff
@@ -0,0 +1,119 @@
+###################
+Credit Card payment
+###################
+
+.. note::
+   Only users with the Project Admin role will be able to view and modify the
+   credit card panel
+
+The Catalyst Cloud now accept payments via credit card. This section will explain how you
+can setup a credit card to pay your invoices.
+
+You and/or your billing contact will still receive a usage summary from us each month with your usage details and a confirmation that your invoice has been paid.
+
+Adding and switching to credit card payment
+===========================================
+
+To setup a credit card, please login_ to the dashboard and follow these steps:
+
+1) Go to Account Details, under Management
+2) Select the Project Billing Details panel
+
+.. image:: resources/billing-details.png
+   :align: center
+
+As you can see from the screenshot, your payment method should be as "Invoice".
+There are a few different ways to setup your credit card payment. You can also add a credit card without changing the payment method.
+
+To change to credit card and to set it up click on Change Payment method:
+
+.. image:: resources/payment-method.png
+   :align: center
+
+
+1) Under Payment Method, select Credit Card 
+2) As you can see you should have the message: "No credit cards added"
+3) Click on "+", this will allow you to add a new credit card
+
+.. image:: resources/add-credit-card.png
+   :align: center
+
+
+1) Add the necessary information
+2) Click on Submit
+3) After it loads, click on Save
+4) After clicking Save, the process may take 2-3 minutes to be completed.
+
+.. image:: resources/payment-method-filled.png
+   :align: center
+
+After adding the credit card, your Project Billing Details should have been updated:
+
+.. image:: resources/billing-details-updated.png
+   :align: center
+
+
+Adding a credit card
+====================
+
+.. note::
+   If you have credit card payment as your selected method. You are not allowed to delete
+   the selected credit card.
+
+You can add multiple credit cards per project. Only the credit card you have selected for the project will be charged.
```

Comment:
You can add multiple credit cards to your account. Only the credit card you have selected for the project will be charged, and different projects can be charged to different credit cards.